tests/test_plugin.py

```diff
@@ -424,17 +428,13 @@ def test_menu_cache_invalidate_after_menucontent_archive(self):
             placeholder, menu_content.menu, "menu/menu.html"
         )
 
-        # Now publish the page content containing the plugin,
-        # so the page can be viewed
-        version = page_content.versions.get()
-        version.publish(self.get_superuser())
-
         # Making sure there is no cachekey existed before rendering page
         cache_key = CacheKey.objects.all().count()
         self.assertEqual(cache_key, 0)
-        # And view the page
```

Comment:
I'm sure that this test should work because the version was published and the endpoint was a published endpoint!

Reply:
# https://github.com/divio/django-cms/blob/2daeb7d63cb5fee49575a834d0f23669ce46144e/cms/tests/test_plugins.py#L160 here we  were trying to add a plugin to a draft page.  test above renders  a draft page after adding the plugin which increases the cachecount. earlier before my changes were added.